docs/source/gbd2019_models/causes/ich/index.rst

```diff
@@ -192,154 +164,79 @@ For example, the *Definitions* table for *SIR* and *With-Condition and Free of C
      - State Name
      - Definition
    * - S
-     - Susceptible
-     - Susceptible to {cause name}
-   * - I
-     - Infected
-     - Infected with {cause name}
-   * - R
-     - Recovered
-     - Infected with {cause name}
-
-
-**With-Condition and Free of Condition Model**
-
-.. list-table:: State Definitions
-   :widths: 1, 5, 10
-   :header-rows: 1
-
-   * - State
-     - State Name
-     - Definition
+     - **S**\usceptible to Intracerebral Hemorrhage
+     - Simulant that has not already had an intracerebral hemorrhage
+   * - A
+     - **A**\cute Intracerebral Hemorrhage
+     - Simulant that is in duration-based period starting day of incidence of a first-ever stroke through day 28 following the event
    * - C
-     - With **C**\ ondition
-     - Born with {cause name}
-   * - F
-     - **F**\ ree of Condition
-     - Born without {cause name}
+     - **C**\hronic Intracerebral Hemorrhage
+     - Simulant that has survived more than 28 days following their last intracerebral hemorrhage and who may be experiencing chronic elevated mortality and disability due to the event. 
 
-Include states, their names and definitions appropriate to your model.
 
 States Data
 """""""""""
 
-This table contains the **measures** and their **values** for each state in cause-model diagram. This information is used to 
-initialize the model. The common measures in each state are prevalence, birth prevalence, excess mortality rate and disability weights. 
-Cause specific mortality rate is the common measure for all states. In most of the models either prevalence or birth prevalence is used. 
-But in some rare cases like neonatal models both prevalence and birth prevalence are used in model initialization. The Value column contains the formula to calculate 
-the measure in each state.
-
-The structure of the table is as below. For each state, the measures and values must be included.
-
-.. list-table:: States Data
-   :widths: 20 25 30 30
+.. list-table:: State Data
+   :widths: 5 10 10 20
    :header-rows: 1
-   
-   * - State
-     - Measure
-     - Value
-     - Notes
-   * - State
-     - prevalence
-     - 
-     - 
-   * - State
-     - birth prevalence
-     - 
-     - 
-   * - State
-     - excess mortality rate
-     - 
-     - 
-   * - State
-     - disabilty weights
-     - 
-     -
-   * - ALL
-     - cause specific mortality rate
-     - 
-     - 
-
-An example of SI model with both prevalence and birth prevalence in the initialization is given below to explain better. 
 
-
-.. list-table:: States Data
-   :widths: 20 25 30 30
-   :header-rows: 1
-   
    * - State
      - Measure
      - Value
      - Notes
+   * - All
+     - cause-specific mortality rate (csmr)
+     - :math:`\frac{\text{deaths_c496}}{\text{population}}`
+     -
+   * - :math:`\text{D}_A`
```

Comment:
these CSMRs will be a little confusing to the engineers; I don't think they need to be included in the table.